rotkehlchen/db/settings.py

```diff
@@ -339,8 +313,8 @@ def db_settings_from_dict(
         elif key == 'address_name_priority':
             specified_args[key] = json.loads(value)
         else:
-            if key == 'eth_rpc_endpoint':
-                continue  # temporary since setting is removed in migration and may still get here
+            if key in {'eth_rpc_endpoint'} | SETTINGS_MOVED:  # temporary since these settings
+                continue  # are moved in migrations/updates and may still get here
```

Comment:
do we load the settings during the upgrade?

Reply:
It happens during the migration and fails in the this test: `rotkehlchen/tests/data_migrations/test_migrations.py::test_migration_1[None-False-0-data_migration_v0.db] `